packages/sanity/src/core/components/commandList/CommandList.tsx

```diff
@@ -539,25 +573,24 @@ export const CommandList = forwardRef<CommandListHandle, CommandListProps>(funct
    */
   useEffect(() => {
     if (autoFocus) {
-      focusElement()
+      focusElement(autoFocus)
     }
-  }, [autoFocus, focusElement])
+  }, [autoFocus, canReceiveFocus, focusListElement, focusInputElement, focusElement])
 
-  // If the input element is present, we want to ensure that the virtual list element is not focusable.
-  // This is to prevent the virtual list from being focused when the input element is focused.
-  const rootTabIndex = inputElement ? -1 : tabIndex
+  const rootTabIndex = canReceiveFocus ? 0 : -1
 
   return (
     <VirtualListBox
-      data-focus-visible={focusVisible}
       id={getCommandListChildrenId()}
+      onMouseEnter={handleVirtualListMouseEnter}
```

Comment:
I wonder if these mouse events should be added to the `VirtualListChildBox` instead to avoid this from happening? 

https://user-images.githubusercontent.com/15094168/235634281-c1cd87ee-88a5-423d-8807-ecaad415a373.mov